saleor/webhook/transport/synchronous/transport.py

```diff
@@ -326,15 +325,9 @@ def trigger_webhook_sync(
     return send_webhook_request_sync(delivery, **kwargs)
 
 
-# TODO: should be called once an app starts, not on each request
 if settings.ENABLE_BREAKER_BOARD:
-    trigger_webhook_sync = BreakerBoard(
-        storage=import_string(settings.BREAKER_BOARD_STORAGE_CLASS_STRING)(),  # type: ignore[arg-type]
-        failure_threshold=settings.BREAKER_BOARD_FAILURE_THRESHOLD_PERCENTAGE,
-        failure_min_count=settings.BREAKER_BOARD_FAILURE_MIN_COUNT,
-        cooldown_seconds=settings.BREAKER_BOARD_COOLDOWN_SECONDS,
-        ttl=settings.BREAKER_BOARD_TTL,
-    )(trigger_webhook_sync)
+    breaker_board = initialize_breaker_board()
+    trigger_webhook_sync = breaker_board(trigger_webhook_sync)
```

Comment:
I wonder if we could apply the breaker board as a decorator:

```
@breaker_bord()
def trigger_webhook_sync(...)
```

This could be possible if we moved the if-condition to the initialization. Then all usages of breaker would be as simple as this.

Reply:
Ain't so simple. Since now initialize_breaker_board can return None, it cannot be used like this.